src/services/auth.js

```diff
@@ -35,3 +35,48 @@ export const googleLoginRequest = async data => {
   setAuthToken(token);
   return decoded;
 };
+
+export const forgotPasswordRequest = async data => {
+  try {
+    const response = await axios.post('api/users/reset', data);
+    return response;
+  } catch (e) {
+    return {
+      status: 400,
+      data: 'Bad email',
+    };
+  }
+};
+
+export const resetApprove = async linkId => {
+  try {
+    const l = encodeURIComponent(linkId);
+    const response = await axios.post(`api/users/reset/${l}`, {});
+    return response;
+  } catch (e) {
+    return {
+      status: 400,
+      data: 'Link is uncorrected',
+    };
```

Comment:
I think you should make classes on backend side to throw errors, and catch them, not like returning default hardcoded objects

Reply:
Because of axios. If error middleware throw error. Axios throw error too. And don't return the right response with an error.

Reply:
Are you sure? I think you should try again.